### ISSUE TITLE
New version release

### ISSUE THREAD

**Issue by berceanu** [sysnetlab/SenSee#2]
Is this app still actively maintained?
Could we get a new ```.apk``` based on the code in this repo? :)

**Comment by huichen-cs** [comment]
Have you tried the APK file in the Google Play store (https://play.google.com/store/apps/details?id=sysnetlab.android.sdc)? Does it work for you?

**Comment by berceanu** [comment]
Yes, but it would be nice to be able to save the sensor output to a database instead of plain text files and potentially sync that database from multiple users to a central server. This would allow running experiments in parallel.

**Comment by huichen-cs** [comment]
Thanks for pointing that out. 

You can save the data to your dropbox. Perhaps, as a work-around, you can create a shared a dropbox account, and everyone syncs to it. 

We have a number of enhancements we want to do; however, did not get a chance to work on it due to other priorities. Those issues are at https://github.com/sysnetlab/SensorDataCollector/issues?q=is%3Aopen+is%3Aissue. 

If you have other requests, you may create issues here. The issue requests will help us prioritize things.
